test/xpu_api/numerics/complex.number/complex.member.ops/plus_equal_complex.pass.cpp

```diff
@@ -30,10 +30,12 @@ test()
     dpl::complex<T> c3;
 
     c3 = c;
```

Comment:
Seems like this line should also be included in the `#if macro` block.

Reply:
Fixed